libgalois/src/PropertyGraph.cpp

```diff
@@ -396,14 +396,14 @@ katana::PropertyGraph::ConstructEntityTypeIDs() {
   // when EntityTypeIDs are not expected in properties then we have nothing to do here
   KATANA_LOG_WARN("Loading types from properties.");
   node_entity_type_manager_ = EntityTypeManager{};
-  EntityTypeIDArray node_entity_type_ids_;
+  node_entity_type_ids_ = EntityTypeIDArray{};
```

Comment:
Minor: Where is `EntityTypeIDArray` defined? Asking because I was using a type alias `EntityTypeIDVec` in PropertyGraph.h

Reply:
https://github.com/KatanaGraph/katana/blob/18d4559310428a82f019472eeb7bd8d7df163222/libgalois/include/katana/PropertyGraph.h#L57

Reply:
Amber, please be mindful of semantic differences b/w naming Array and Vec. Use Array when it is not resize-able and Vec when it can be resized.